hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/opt/physical/SortPhysicalRel.java

```diff
@@ -59,6 +65,43 @@ public boolean requiresSort() {
         return requiresSort;
     }
 
+    @Override
+    @SuppressWarnings("checkstyle:magicnumber")
+    public RelOptCost computeSelfCost(RelOptPlanner planner, RelMetadataQuery mq) {
+        // Copy of org.apache.calcite.rel.core.Sort.computeSelfCost but includes our requiresSort flag.
```

Comment:
Add the note [to that file](https://github.com/hazelcast/hazelcast/blob/master/NOTICE) about used code, please

Reply:
That file already contains info abount whole packages:
```
The packages:
com.hazelcast.sql
com.hazelcast.jet.sql

contain code originating from the Apache Calcite (https://github.com/apache/calcite)
```

Should I add also that class explicite?

Reply:
Hmm, I guess, it is covered. Thanks.